aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -13,7 +13,56 @@ uid: fundamentals/servers/kestrel
 
 By [Tom Dykstra](https://github.com/tdykstra), [Chris Ross](https://github.com/Tratcher), and [Stephen Halter](https://twitter.com/halter73)
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
+
+Kestrel is a cross-platform [web server for ASP.NET Core](xref:fundamentals/servers/index). Kestrel is the web server that's included and enabled by default in ASP.NET Core project templates.
+
+Kestrel supports the following scenarios:
+
+* HTTPS
+* [HTTP/2](xref:fundamentals/servers/kestrel/http2) (except on macOS&dagger;)
```

Comment:
HTTP/2 doesn't work on macOS because of no HTTPS?

Reply:
Correct. There's no ALPN support on the server: https://github.com/dotnet/runtime/issues/27727